### PR TITLE
Add iPython notebook for importing/exporting dynamodb data.

### PR DESCRIPTION
**Pull request by wetchler**
This is only a temporary solution, but I didn't want to put a lot of effort into making clean libraries and whatnot for the time being. Eventually this should probably be a rake task to save or load an entire database.

### CODE REVIEW

**Comment by wetchler** [comment]
Fixes #24

File: notebooks/DynamoDbImportExport.ipynb [hunk 1]
```diff
@@ -0,0 +1,495 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "import boto3\n",
```
Comment (AlexNisnevich) [inline notebooks/DynamoDbImportExport.ipynb:92]: I had to `pip install boto3` to get this working locally. Should this notebook be containerized somehow?

File: notebooks/DynamoDbImportExport.ipynb [hunk 2]
```diff
@@ -0,0 +1,495 @@
...
+    "# If these are left blank, we'll try to find environment\n",
+    "# variables by the same name and use those.\n",
+    "AWS_ACCESS_KEY_ID = ''\n",
+    "AWS_SECRET_ACCESS_KEY = ''\n",
+    "AWS_REGION = ''\n",
+    "\n",
+    "# If this is true, we connect to a dynamodb-local instance\n",
+    "# running on port 8000. The AWS keys above are then\n",
+    "# irrelevant, unless you changed it to not run as a --sharedDb.\n",
+    "USE_DYNAMODB_LOCAL = True"
```
Comment (AlexNisnevich) [inline notebooks/DynamoDbImportExport.ipynb:53]: `USE_DYNAMODB_LOCAL` and `set_local_db()` are not used anywhere in the rest of the notebook.

File: notebooks/DynamoDbImportExport.ipynb [hunk 3]
```diff
@@ -0,0 +1,495 @@
...
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# Set either RUN_IMPORT or RUN_EXPORT to true.\n",
+    "# Here we set both for the sake of example to see\n",
+    "# them both in action.\n",
+    "RUN_EXPORT = True\n",
```
Comment (AlexNisnevich) [inline notebooks/DynamoDbImportExport.ipynb]: `RUN_EXPORT` and `RUN_IMPORT` should probably be set to False by default to prevent accidents.

**Comment by AlexNisnevich** [comment]
When I run this notebook (using local dynamodb), everything is fine until I get to the last cell, and then I get:

```
Loading 27146 bytes of pickled data from dynamodb.pickle
Loaded data for 13 tables
         ursus_development_changedfields:       0 items        0 kB
    ursus_development_involved_civilians:       5 items        2 kB
             ursus_development_screeners:       5 items        0 kB
                ursus_development_visits:       0 items        0 kB
             ursus_development_feedbacks:       0 items        0 kB
             ursus_development_incidents:       5 items        1 kB
                 ursus_development_users:       1 items        0 kB
         ursus_development_general_infos:       5 items        1 kB
          ursus_development_globalstates:       1 items        0 kB
                ursus_development_events:       0 items        0 kB
          ursus_development_auditentries:       0 items        0 kB
        ursus_development_agencystatuses:       1 items        0 kB
     ursus_development_involved_officers:       5 items        1 kB
Importing...
** Importing into table ursus_development_changedfields **
(u'ursus_development_changedfields', 'already exists, deleting it first and starting afresh.')
Delete operation sent. Waiting until table no longer exists.
---------------------------------------------------------------------------
ParamValidationError                      Traceback (most recent call last)
<ipython-input-11-dc7f9bcba694> in <module>()
     26         schema = clean_up_table_schema(info["Table"], rename_prefix=IMPORT_TABLES_ADD_PREFIX)
     27         items = info["Items"]
---> 28         client.create_table(**schema)
     29         print("Create operation sent. Waiting until table exists.")
     30         tbl = resource.Table(new_name)

/Users/alex/anaconda/lib/python2.7/site-packages/botocore/client.pyc in _api_call(self, *args, **kwargs)
    157                     "%s() only accepts keyword arguments." % py_operation_name)
    158             # The "self" in this scope is referring to the BaseClient.
--> 159             return self._make_api_call(operation_name, kwargs)
    160 
    161         _api_call.__name__ = str(py_operation_name)

/Users/alex/anaconda/lib/python2.7/site-packages/botocore/client.pyc in _make_api_call(self, operation_name, api_params)
    468         }
    469         request_dict = self._convert_to_request_dict(
--> 470             api_params, operation_model, context=request_context)
    471 
    472         handler, event_response = self.meta.events.emit_until_response(

/Users/alex/anaconda/lib/python2.7/site-packages/botocore/client.pyc in _convert_to_request_dict(self, api_params, operation_model, context)
    521 
    522         request_dict = self._serializer.serialize_to_request(
--> 523             api_params, operation_model)
    524         prepare_request_dict(request_dict, endpoint_url=self._endpoint.host,
    525                              user_agent=self._client_config.user_agent,

/Users/alex/anaconda/lib/python2.7/site-packages/botocore/validate.pyc in serialize_to_request(self, parameters, operation_model)
    268                                                     operation_model.input_shape)
    269             if report.has_errors():
--> 270                 raise ParamValidationError(report=report.generate_report())
    271         return self._serializer.serialize_to_request(parameters,
    272                                                      operation_model)

ParamValidationError: Parameter validation failed:
Unknown parameter in ProvisionedThroughput: "LastIncreaseDateTime", must be one of: ReadCapacityUnits, WriteCapacityUnits
Unknown parameter in ProvisionedThroughput: "LastDecreaseDateTime", must be one of: ReadCapacityUnits, WriteCapacityUnits
```

File: notebooks/DynamoDbImportExport.ipynb (hunk 3 above)
Comment (wetchler) [reply notebooks/DynamoDbImportExport.ipynb]: done

File: notebooks/DynamoDbImportExport.ipynb (hunk 2 above)
Comment (wetchler) [reply notebooks/DynamoDbImportExport.ipynb:53]: They are now

**Review by AlexNisnevich — APPROVED** [review]
LGTM

**Pull request merged**